[{'domain': '.weibo.cn', 'path': '/', 'value': '1502377324', 'secure': False, 'httpOnly': False, 'name': 'SSOLoginState'}, {'domain': '.weibo.cn', 'expiry': 1533913324.699964, 'path': '/', 'value': '0XDlCKIhViYDuV', 'name': 'SUHB', 'httpOnly': False, 'secure': False}, {'domain': '.weibo.cn', 'expiry': 1533913324.699279, 'path': '/', 'value': '_2A250iAU8DeRhGeBM4lcS8SjOyTyIHXVUcqt0rDV6PUJbkdANLVPEkW08uKD-QFqdKaByROmX9oxOdLXQvQ..', 'name': 'SUB', 'httpOnly': True, 'secure': False}, {'domain': '.weibo.cn', 'expiry': 1817737324.509334, 'path': '/', 'value': 'AgZEwYOkpufNKvhLhWeyaOiVsGw2wwdPUgQ1Eylmgd47cdGKWA4P-fix_Bq1U6Rkv4AU9qt-Fj8QM0itsOgf7Ro.', 'name': 'SCF', 'httpOnly': True, 'secure': False}, {'domain': '.weibo.cn', 'expiry': 1504969314.375669, 'path': '/', 'value': '5482930b7c7fd842ae1cc30b03a92d26', 'name': '_T_WM', 'httpOnly': True, 'secure': False}]
判断页面1成功 0失败  结果是=1
[<selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-1")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-2")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-3")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-4")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-5")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-6")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-7")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-8")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-9")>, <selenium.webdriver.remote.webelement.WebElement (session="42d56e46da93d42e80bee7f53296d8de", element="0.6680519028904397-10")>]
伦哥：好棒！第1次运行了
运行时间：2017-08-10 23:02:06
1
False
2
False
3
False
4
False
5
False
6
False
7
False
8
False
9
Traceback (most recent call last):
  File "my_repost.py", line 192, in <module>
    repost(driver1)
  File "my_repost.py", line 164, in repost
    result = EC.alert_is_present()(browser)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/support/expected_conditions.py", line 339, in __call__
    alert.text
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/common/alert.py", line 69, in text
    return self.driver.execute(Command.GET_ALERT_TEXT)["value"]
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 254, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/remote_connection.py", line 464, in execute
    return self._request(command_info[0], url, body=data)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/remote_connection.py", line 488, in _request
    resp = self._conn.getresponse()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 1174, in getresponse
    response.begin()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 282, in begin
    version, status, reason = self._read_status()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 243, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/socket.py", line 571, in readinto
    return self._sock.recv_into(b)
ConnectionResetError: [Errno 54] Connection reset by peer
